mtp_api/apps/security/views.py

```diff
@@ -586,3 +588,21 @@ def reject(self, request, pk):
         serializer.is_valid(raise_exception=True)
         check = serializer.reject(by=request.user)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class CheckAutoAcceptRuleView(
+    mixins.CreateModelMixin,
+    mixins.ListModelMixin,
+    mixins.UpdateModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = CheckAutoAcceptRule.objects.all()
+    filter_backends = (DjangoFilterBackend, filters.OrderingFilter)
+    serializer_class = CheckAutoAcceptRuleSerializer
+    ordering_fields = ('created',)
+    ordering = ('created',)
+    permission_classes = (
+        IsAuthenticated,
+        SecurityCheckPermissions,
```

Comment:
So we're reusing the `SecurityCheckPermissions` permission class here.

I'm not that familiar with this, would this use the `CheckAutoAcceptRule`'s permissions because `SecurityCheckPermissions` inherits from `ActionsBasedPermissions` (through `SecurityProfilePermissions`)?